doc/classes/Spatial.xml

```diff
@@ -287,6 +287,7 @@
 			<argument index="0" name="offset" type="Vector3">
 			</argument>
 			<description>
+				Changes the node's position by given offset [Vector3] in local space.
```

Comment:
Missing "a" or "the" before "given"?

Reply:
This was a copy-paste from an above description. Ill add a "the" to both.